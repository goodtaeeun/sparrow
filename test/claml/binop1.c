int main(int argc, char *argv[]) {
  char **pp = argv + 1;
  sparrow_print(pp);
}
