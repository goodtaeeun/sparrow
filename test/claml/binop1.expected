/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "binop1.c"
int main(int argc, char *argv[]) {
  char **pp;

  {
    {
#line 2
      pp = argv + 1;
#line 3
      sparrow_print(pp);
    }
    return (0);
  }
}
